sys-devel/lld-toolchain-symlinks/lld-toolchain-symlinks-16.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit multilib
+
+DESCRIPTION="Symlinks to use LLD on binutils-free system"
+HOMEPAGE="https://wiki.gentoo.org/wiki/Project:LLVM"
+SRC_URI=""
+S=${WORKDIR}
+
+LICENSE="public-domain"
+SLOT="${PV}"
+KEYWORDS=""
+PROPERTIES="live"
+IUSE="+native-symlinks"
+
+RDEPEND="
+	sys-devel/lld
+"
+
+src_install() {
+	use native-symlinks || return
+
+	local dest=/usr/lib/llvm/${SLOT}/bin
+	dodir "${dest}"
+	dosym ../../../../bin/ld.lld "${dest}/ld"
+	for abi in $(get_all_abis); do
```

Comment:
make `local`?

Reply:
Right. Got lost when I removed `for t` after copy-pasting ;-).